src/App.tsx

```diff
@@ -1,34 +1,81 @@
-/* eslint-disable max-len */
-import React from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [query, setQuery] = useState<string>('');
+  const [originalTodos, setOriginalTodos] = useState<Todo[]>([]);
+
+  useEffect(() => {
+    getTodos().then(todosFromServer => setOriginalTodos(todosFromServer));
```

Comment:
Also, you need to add error handling
```suggestion
    getTodos().then(setOriginalTodos);
```